be/src/vec/functions/function_datetime_string_to_string.h

```diff
@@ -41,20 +41,81 @@
 #include "vec/functions/date_time_transforms.h"
 #include "vec/functions/function.h"
 
+#define GET_COLUMN_CONST_RETURN_IF_ERROR(Index, ColumnType)                                                         \
+    const IColumn& source_col##Index = *block.get_by_position(arguments[Index]).column;                 \
+    const auto* delta_const_column##Index = typeid_cast<const ColumnType*>(&source_col##Index);        \
+    if (!delta_const_column##Index) {                                                                   \
+        return Status::InternalError("Illegal column " +                                                \
+                block.get_by_position(arguments[Index]).column->get_name() + "(index=" +                \
+                std::to_string(Index) + ", type=" + typeid(source_col##Index).name() +                  \
+                                     ") is not const " + name);                                         \
+    }                                                                                                   \
+
 namespace doris {
 class FunctionContext;
 } // namespace doris
 
 namespace doris::vectorized {
 
 template <typename Transform>
-class FunctionDateTimeStringToString : public IFunction {
+class FunctionDateTimeStringToStringOneArg : public IFunction {
+public:
+    static constexpr auto name = Transform::name;
+    static constexpr bool has_variadic_argument =
+            !std::is_void_v<decltype(has_variadic_argument_types(std::declval<Transform>()))>;
+    static FunctionPtr create() { return std::make_shared<FunctionDateTimeStringToStringOneArg>(); }
+    String get_name() const override { return name; }
+    size_t get_number_of_arguments() const override { return 0; }
+    bool is_variadic() const override { return true; }
+    DataTypes get_variadic_argument_types_impl() const override {
+        if constexpr (has_variadic_argument) return Transform::get_variadic_argument_types();
```

Comment:
this will break format rule. please use builtin clang-format file to format your pr.